src/async/tasks.md

```diff
@@ -0,0 +1,64 @@
+# Tasks
+
+Runtimes have the concept of a "Task", similar to a thread but much
+less resource-intensive.
+
+A Task has a single top-level Future which the executor polls to make progress.
+That future may have one or more nested futures that its `poll` method polls,
+corresponding loosely to a call stack. Concurrency within a task is possible by
+polling multiple child futures, such as racing a timer and an I/O operation.
+
+```rust,compile_fail
+use tokio::io::{self, AsyncReadExt, AsyncWriteExt};
+use tokio::net::TcpListener;
+
+#[tokio::main]
+async fn main() -> io::Result<()> {
+    let listener = TcpListener::bind("127.0.0.1:6142").await?;
+	println!("listening on port 6142");
+
+    loop {
+        let (mut socket, addr) = listener.accept().await?;
+
+        println!("connection from {addr:?}");
+
+        tokio::spawn(async move {
```

Comment:
This is the first time async blocks appear (also alluded to in async-await and pin sections). They probably merit a sentence somewhere.

Reply:
That's mentioned in the speaker notes. In general, it's a non-goal to put all explanation on the slide itself.

Reply:
Ah, this was a failure in my search. I agree the note in speaker notes is good and sufficient.

For context, I noticed `async move` later and was trying to find where async blocks was explained/mentioned. Both "async" and "block" appear a lot, so I searched for `async block`, `async {`, or `async move`. (I don't think there's anything actionable here.)